pkg/aws/helpers.go

```diff
@@ -864,6 +864,13 @@ func GetResourceIdFromARN(stringARN string) (string, error) {
 	return parsedARN.Resource[index+1:], nil
 }
 
+func MockOidcConfig(id string, issuerUrl string) (*cmv1.OidcConfig, error) {
```

Comment:
@hunterkepley If this is only used for tests, then I'd suggest we create `test_helpers.go` file for clarity 👍